frontend/src/components/TransferCreditHolder.tsx

```diff
@@ -68,6 +76,58 @@ const Wrapper = styled.div`
   height: 100%;
 `;
 
+/**
+ * Determines the corresponding transferable exam type and name for the given NEU course.
+ * @param examGroups The transferable exam groupings (either AP or IB).
+ * @param course The NEU course to query against.
+ * @returns The exam type and name as a string.
+ */
+function findExamText(
+  examGroups: TransferableExamGroup[],
+  course: ScheduleCourse
+) {
+  for (const examGroup of examGroups) {
+    for (const transferableExam of examGroup.transferableExams) {
+      for (const mappableCourse of transferableExam.mappableCourses) {
+        if (
+          mappableCourse.subject === course.subject &&
+          String(mappableCourse.classId) === course.classId
+        ) {
+          return transferableExam.type + " " + transferableExam.name;
+        }
+      }
+    }
+  }
+  // If not found, return an empty string.
+  return "";
+}
+
+/**
+ * Gets the tooltip hover text for the transfer credits container.
+ * @param course The NEU course to query against.
+ * @returns The tooltip hover text as a string.
+ */
+function getTooltipText(course: ScheduleCourse) {
+  const apExamText = findExamText(APExamGroups2020To2021, course);
+  const ibExamText = findExamText(IBExamGroups2020To2021, course);
+
+  if (apExamText === "" && ibExamText === "") {
+    return "Transferred as " + course.subject + course.classId;
```

Comment:
talked to @justineluo about this. Let's keep a space between the subject and the class ID. Also use string interpolation for this :))